cmd/nvidia-ctk/cdi/generate/generate.go

```diff
@@ -93,7 +100,13 @@ func (m command) build() *cli.Command {
 		},
 	}
 
-	c.Flags = []cli.Flag{
+	flags = []cli.Flag{
+		&cli.StringFlag{
+			Name:        "config",
+			Usage:       "Specify the path to the config file to use",
+			Destination: &opts.config,
+			Sources:     cli.EnvVars("NVIDIA_CTK_CONFIG_FILE"),
+		},
```

Comment:
It almost feels as if we need to set this as part of the `nvidia-ctk` top-level command and then pass it in to the subcommand constructor. Otherwise we're going to have to reimplement this for every subcommand.